docs/vault/guide.md

```diff
@@ -39,27 +72,131 @@ Following the installation instructions, the Vault client will submit new BTC bl
 
 By default, Vault client will try to accept replace requests by other Vaults to take over their locked BTC. Accepting replace requests increases the relative share of locked BTC for the Vault and hence increases the earned fees and block rewards of the Vault client.
 
-## Collateral
+## Upgrading the Vault Client
+
+The Vault client is constantly being improved. Follow this guide to get the latest version of the Vault client.
+
+We will announce on public channels when a new release is made available for the Vault client. The changelog and binaries will be published on the [release page](https://github.com/interlay/interbtc-clients/releases). Depending on the method of installation:
+
+!> Occasionally, breaking changes will be introduced to the Vault client. In these cases it is important to update your Vault client in a timely manner to not run the risk of being slashed. We will notify on our channels (Discord, Twitter, Telegram) when this is the case.
+
+### Standard Installation
```

Comment:
Missing interlay tabs

Reply:
True, this is just copy paste from before. But fixed it now.